docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,124 @@
+communication between Aiven Management and monitoring systems and your private workloads.
+The bastion proxy service is also responsible for facilitating a communication channel that
+allows the Aiven Management plane to launch services within your private cloud.
+
+### Traffic to the bastion
+
+Aiven leverages redundant gateways to conduct BYOC operations. Ingress traffic from
+particular Aiven gateway IP addresses to the bastion subnet on specific ports is required.
+Each gateway is hosted on a separate and geographically remote cloud provider to protect
+against communication failures.
+
+| Source                 | Destination    | Description                                                                                                                                   |
+| ---------------------- | ---------------| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aiven Management plane | Bastion subnet | Aiven uses SSH from the Aiven Management plane to the bastion service node for troubleshooting, setup, or configuration tasks on the bastion. |
+| Aiven Management plane | Bastion subnet | Proxy service is used to tunnel the traffic from the Aiven Management plane to workload nodes.                                                |
+| Aiven Management plane | Bastion subnet | Aiven Management plane accesses the bastion service node to retrieve status information and perform routine operations.                       |
+
+### Traffic from the bastion
+
+| Source         | Destination      | Description                                                                                                                             |
+| -------------- | -----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet | Aiven Management | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes              |
+| Bastion subnet | Aiven Management | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                                 |
+| Bastion subnet | AWS CloudFront   | Communication channel for downloading RPM packages for bastion and workload functionality and maintenance                               |
+| Bastion subnet | Workload subnet  | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet | Workload subnet  | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Bastion subnet | DNS & NTP        | Destination dependant on cloud provider                                                                                                 |
```

Comment:
Same here, could even specify source and destination ports where it makes sense

Reply:
Without port numbers, some descriptions look a bit redundant. Eg. compare lines 38 and 39.

Reply:
See https://github.com/aiven/aiven-docs/pull/298/files#r1657455784